src/uwtools/drivers/esg_grid.py

```diff
@@ -0,0 +1,100 @@
+"""
+A driver for esg_grid.
+"""
+
+from pathlib import Path
+from typing import Optional
+
+from iotaa import asset, dryrun, task, tasks
+
+from uwtools.config.formats.nml import NMLConfig
+from uwtools.drivers.driver import Driver
+from uwtools.strings import STR
+
+
+class EsgGrid(Driver):
```

Comment:
I'm sorry to make this comment so late in the game, but I think that maybe `ESGGrid` would be the commonly preferred form for this name, given that "ESG" ("Extended Schmidt Gnomonic" -- I just looked it up) is an acronym. Similarly, our JEDI driver is called `JEDI` and not `Jedi`, because it's an acronym; or `FV3` and not `Fv3`. I don't think we should change things like `esg_grid` or `esg-grid`, but probably `ESGGrid` as a class name and `ESG Grid` as a human-text descriptor when referring to the tool and not the Python class, would be appropriate.

Reply:
I'll work on switching this over. Feel free to point out any I miss in my later commits.